litmus-portal/frontend/src/components/ProgressBar/LinearProgressBar/index.tsx

```diff
@@ -1,35 +1,95 @@
-import { Line } from 'rc-progress';
 import React from 'react';
-import { useTheme } from '@material-ui/core/styles';
+import { withStyles, lighten, LinearProgress, Theme } from '@material-ui/core';
 
 interface LinearProgressBarProps {
   value: number | number[];
   isDefault?: boolean;
   width: number;
 }
+const lightConstant = 0.5;
+
+const RedLinearProgress = withStyles((theme: Theme) => ({
+  bar: {
+    backgroundColor: theme.palette.error.dark,
+  },
+  root: {
+    backgroundColor: lighten(theme.palette.error.dark, lightConstant),
+  },
+}))(LinearProgress);
+
+const YellowLinearProgress = withStyles((theme: Theme) => ({
+  bar: {
+    backgroundColor: theme.palette.warning.main,
+  },
+  root: {
+    backgroundColor: lighten(theme.palette.warning.main, lightConstant),
+  },
+}))(LinearProgress);
+
+const GreenLinearProgress = withStyles((theme: Theme) => ({
+  bar: {
+    backgroundColor: theme.palette.primary.dark,
+  },
+  root: {
+    backgroundColor: lighten(theme.palette.primary.dark, lightConstant),
+  },
+}))(LinearProgress);
 
 const LinearProgressBar: React.FC<LinearProgressBarProps> = ({
```

Comment:
This entire code modification here is basically repeating the same file changes done in Analytics Progress Bar. Replace all this logic with a single check based on result value to switch color directly in the component, not by if checks, as mentioned above.